File: LogiCode-Client/src/Class/Course.ts
import { Lesson } from "./Lesson";

export enum eLevel{
    easy,
    meduim,
    hard
}
export class Course{
    constructor(public Id?:number, public Title?:string, public Descreption?:string,
         public ProgrammingLanguage?:string, public DurationHours?:number, public Level?:eLevel, 
        public Price?:number, public ImageUrl?:string, public Points?:Array<string>, public Lessons?:Array<Lesson>){}
}

File: LogiCode-Server/core/DTO/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.DTO
{
    public enum eLevel{easy=1,medןum=2,hard=4}
    public class Course
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string ProgrammingLanguage { get; set; }
        public int DurationHours { get; set; }
        public decimal Price { get; set; }
        public eLevel DurationMinutes { get; set; }
        public string ImageUrl { get; set; }
        public List<string> Points {  get; set; } 
        public List<Lesson> Lessons { get; set; }


    }
}
